.github/workflows/build.yml

```diff
@@ -0,0 +1,35 @@
+name: C/C++ CI
+
+on:
+  push:
+      branches: [ master ]
+  pull_request:
+      branches: [ master ]
+
+jobs:
+  build-and-test:
+      runs-on: ubuntu-latest
+      steps:
+      - uses: actions/checkout@v2
+      - name: install thirdparties
+      run: |
+        sudo apt install -y cmake 
+        sudo apt install -y libboost-filesystem-dev libboost-test-dev libboost-date-time-dev libboost-program-options-dev
+        sudo apt install -y libhdf5-dev
+        sudo apt install -y libeigen3-dev
+        sudo apt install -y libqt4-opengl-dev
+        mkdir -p deps
+        docker pull itisfoundation/iseg-ubuntu-vtk:8.2
+        docker create --name vtkcontainer itisfoundation/iseg-ubuntu-vtk:8.2 bash
+        docker cp vtkcontainer:/work deps/vtk
+        docker pull itisfoundation/iseg-ubuntu-itk:5.0.1
+        docker create --name itkcontainer itisfoundation/iseg-ubuntu-itk:5.0.1 bash
```

Comment:
minor: you might have a ```make install``` that does this? and conversely a ```make clean/uninstall```.
it's a bit easier to reproduce issues from the CI locally than looking into this file to run the same steps

Reply:
yes, I agree. for now I managed to use [`act`](https://github.com/nektos/act) to run the action locally.